src/util.cpp

```diff
@@ -758,15 +791,35 @@ void ArgsManager::ReadConfigStream(std::istream& stream)
     {
         std::string strKey = std::string("-") + it->string_key;
         std::string strValue = it->value[0];
+
         if (InterpretNegatedOption(strKey, strValue)) {
             m_config_args[strKey].clear();
         } else {
             m_config_args[strKey].push_back(strValue);
         }
+
+        // Check that the arg is known
```

Comment:
Can you move this to a function to avoid duplicate code?

Reply:
If you do so then I think you can inline `GetArgWithoutNet` there.

Reply:
Made it into a function and inlined `GetArgWithoutNet`.